.github/workflows/internal-archive-release.yml

```diff
@@ -49,7 +54,8 @@ env:
   PYTHON_TARGET_VERSION: 3.8
   NOTIFICATION_PREFIX: "[Internal Archive Release]"
   TEMP_PROFILE_NAME: "temp_aws_profile"
-  HATCH_ADAPTERS: '["postgres"]' # Must be valid JSON
+  HATCH_ADAPTERS: (postgres) # Must be | deliminated list
+  SKIP_TEST_ADAPTERS: (spark|trino|databricks)
```

Comment:
Minor nit, but I read this a few times thinking that these were "test" adapters. Maybe `SKIP_TESTS_ADAPTERS`? `ADAPTERS_SKIP_TESTS`? Or simply `SKIP_TESTS`? It doesn't really matter in the end, but it caused me to get hung up initially.

Reply:
Renamed to `ADAPTERS_THAT_SKIP_TESTS` for ease of reading. No need to fear Verbosity in the naming, especially when it's only used in one check, methinks.